scripts/send_application_reminder.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+import argparse
+import sys
+from datetime import datetime
+from datetime import timedelta
+
+sys.path.insert(1, ".")
+
+from external_services.exceptions import NotificationError
+import api.routes.application.helpers as helpers
+from app import app
+from config import Config
+from db.models.applications import ApplicationsMethods
+from db.models.forms import FormsMethods
+from external_services.models.notification import Notification
+from flask import current_app
+
+
+def application_deadline_reminder(fund_id: str, round_id: str):
+
+    current_date_time = datetime.now()
+
+    fund_rounds = helpers.get_data(
```

Comment:
Can this be "fund_round" instead of plural (as there is only one fund round returned right?)

Reply:
has been changed now